### PR TITLE
fix(ui): systray format

### LINKED ISSUE #419: [macos] [Systray icon] GPU not included in est. earnings. GPU hash rate not formated
Estimated earnings only includes CPU.

GPU hashrate shows as 45273343 H/s

### PR DESCRIPTION
**Pull request by karczuRF**
Description
Fix system data formatting

Motivation and Context
Resolves #419 

How Has This Been Tested?
Manually

What process can a PR reviewer use to test or verify this change?
**MUST BE YET TESTED ON WINDOWS**

Breaking Changes
---

- [x] None
- [ ] Requires data directory on base node to be deleted
- [ ] Requires hard fork
- [ ] Other - Please specify

### CODE REVIEW

File: src-tauri/src/systemtray_manager.rs [hunk 1]
```diff
@@ -101,19 +107,23 @@ impl SystemtrayManager {
...
+                    Formatter::new().with_decimals(2).with_separator("").format(data.estimated_earning / 1_000_000.0)
                 )
             }
             CurrentOperatingSystem::Linux => "Not supported".to_string(),
             CurrentOperatingSystem::MacOS => {
                 format!(
-                    "CPU:\n  Hashrate: {:.0} H/s\n  Usage: {:.0}%\nGPU:\n  Hashrate: {:.0} H/s\n  Usage: {:.0}%\nEst. earning: {} tXTM/day",
-                    data.cpu_hashrate, data.cpu_usage, data.gpu_hashrate, data.gpu_usage, format_balance(data.estimated_earning)
+                    "CPU:\n  Hashrate: {} H/s\n  Usage: {:.0}%\nGPU:\n  Hashrate: {} H/s\n  Usage: {:.0}%\nEst. earning: {} tXTM/day",
+                    Formatter::new().with_decimals(2).with_separator("").format(data.cpu_hashrate),
```
Comment (stringhandler) [inline src-tauri/src/systemtray_manager.rs:122]:
```suggestion
                    Formatter::new().with_decimals(0).with_separator("").format(data.cpu_hashrate),
```
I think 0 decimals is sufficient

**Review by stringhandler — APPROVED** [review]

**Review by brianp — CHANGES_REQUESTED** [review]
image

Estimated earnings per day has quotes around it, and only seems to be calculated based on CPU, not combined with GPU.

**Review by brianp — APPROVED** [review]
Looks much better 👍🏻 

image

**Comment by Tas4tari** [comment]
Tested on Windows 11, app version 0.5.60 and the sys tray format appears cut off
Screenshot (182)

**Pull request merged**